packages/esm-patient-registration-app/src/patient-registration/input/dummy-data/dummy-data-input.component.tsx

```diff
@@ -23,16 +23,18 @@ export const dummyFormValues: FormValues = {
   monthsEstimated: 2,
   birthdateEstimated: true,
   telephoneNumber: '0800001066',
-  address1: 'Bom Jesus Street',
-  address2: '',
-  cityVillage: 'Recife',
-  stateProvince: 'Pernambuco',
-  country: 'Brazil',
-  postalCode: '50030-310',
   isDead: false,
   deathDate: '',
   deathCause: '',
   relationships: [],
+  address: {
```

Comment:
@vasharma05 , does this assume the address fields are static, hardcoded?

Reply:
No @rbuisson, these are just the default values to these fields. Other fields can be added dynamically, if they are present in the form, handled by Formik.